endpoints-support.md

```diff
@@ -69,6 +69,12 @@
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PATCH    | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/fcoe-networks/{id}</sub>                                                      | DELETE   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+
+|     **Interconnects**                                                                                                                               |
+|<sub>/rest/interconnects/{id}</sub>                                                        | PATCH    | :white_check_mark:  | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |:white_check_mark:   | :white_check_mark: | :white_check_mark:  |
```

Comment:
have you added all the endpoints supportedfor Interconnects? interconnect configuration is missing...verify all others as well.

Reply:
@soodpr Done